### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by davidpage2022**
Hi @brennen12

Could you please review my prac 6 for any naming issues, confusing comments or anything that might need improvement. Thanks.

Note: I used the "while True" error checking technique in guitars.py that Lindsay showed at the end of the "Big demo" video. It looks wrong, but he said we are allowed to do that in this case.

### CODE REVIEW

File: prac_06/guitars.py [hunk 1]
```diff
@@ -1,12 +1,55 @@
...
+    """Gets a non-blank string from the user with prompt message.
+    Continues to ask until valid input is given."""
+    string = input(prompt).strip()
+    while string == "":
+        print("Cannot be blank")
+        string = input(prompt).strip()
+    return string
+
+
+def get_positive_integer(prompt):
```
Comment (Brennen12) [inline prac_06/guitars.py:35]: This is good but I don't think it was asked for.

File: prac_06/guitars.py [hunk 2]
```diff
@@ -2,26 +2,25 @@
...
-    cost = get_positive_float("Cost: $")
+    print("My guitars!")
+    name = input("Name: ").strip()
+    while name != "":
+        year = get_positive_integer("Year: ")
+        cost = get_positive_float("Cost: $")
 
+        guitar = Guitar(name, year, cost)
+        guitars.append(guitar)
+        print(f"{guitar} added.")
```
Comment (Brennen12) [inline prac_06/guitars.py:24]: Does this match the output they asked for exactly?

Comment (davidpage2022) [reply prac_06/guitars.py:24]: Hi @Brennen12 . It actually does! `print(f"{guitar}")` prints the guitar object using the `__str__` method.

File: prac_06/guitars.py (hunk 1 above)
Comment (davidpage2022) [reply prac_06/guitars.py:35]: True, we didn't need input error checking for this exercise. I just copied these from a previous exercise 😄

**Pull request merged**